solr/views.py

```diff
@@ -62,7 +62,10 @@ def cleanup_solr_request(payload):
         if 'fl' not in payload:
             payload['fl'] = 'id'
         else:
-            fields = [x.strip() for x in payload['fl'][0].split(',')]
+            fields = []
```

Comment:
hi Jonny, this works (you have fixed it for the clients), but we are getting away from how solr treats those parameters - the fl parameter is supposed to be a string 'field,fieldb,field....' - if clients send fl as an array, it is actually the client's problem. The reason being: one should be able to copy and paste url params and paste them directly into solr and it should work without the proxy that 'corrects' the parameters

for other fields, such as 'q', even if the client can send array - we should not try to concatenate the two into one 'q'